.github/workflows/integration-tests.yml

```diff
@@ -21,6 +21,8 @@ jobs:
     name: Run integration tests - ${{ matrix.environment }}
     runs-on: ${{ matrix.os }}
     strategy:
+      # avoid cancellation of in-progress jobs if any matrix job fails
+      fail-fast: false
       matrix:
         python-version: [ 3.9 ]
         os: [ "ubuntu-latest" ]
```

Comment:
Could you please add `cloud-staging` as well in below line? (I am not able to comment on a line that has not changed)

Reply:
Done via https://github.com/Qiskit/qiskit-ibm-runtime/pull/135/commits/44a9ae9904c7248e7918f1adea698829c5bc37d9